server/manager/src/main/java/org/apache/accumulo/manager/upgrade/Upgrader12to13.java

```diff
@@ -129,8 +129,20 @@ private static void addCompactionsNode(ServerContext context) {
   }
 
   private void createFateTable(ServerContext context) {
-    ZooKeeperInitializer zkInit = new ZooKeeperInitializer();
-    zkInit.initFateTableState(context);
+
+    if (context.tableOperations().exists(AccumuloTable.FATE.tableName())) {
+      LOG.info("Fate table already exists");
+      return;
+    }
+
+    try {
+      ZooKeeperInitializer zkInit = new ZooKeeperInitializer();
+      zkInit.initFateTableState(context);
+    } catch (Exception e) {
+      if (e.getCause() instanceof KeeperException.NodeExistsException) {
+        LOG.debug("Fate table node already exists in ZooKeeper");
+      }
+    }
```

Comment:
@dlmarion This is equivalent to:

```java
    } catch (KeeperException.NodeExistsException e) {
      LOG.debug("Fate table node already exists in ZooKeeper");
    } catch (Exception e) {
      // completely swallow and ignore
    }
```

Are you sure this is the intended behavior? It looks very sketchy to me.

Reply:
Great catch!

Reply:
Fixed in #5409